app/controllers/majors_controller.rb

```diff
@@ -8,26 +8,26 @@ def self.registered(app)
           app.namespace '/v1/majors' do
             get do
               resp = {
-                message: "This is the majors endpoint.",
-                version: "1.0.0",
-                docs: "https://docs.umd.io/majors",
-                endpoints: ["/list"]
+                message: 'This is the majors endpoint.',
+                version: '1.0.0',
+                docs: 'https://docs.umd.io/majors',
+                endpoints: ['/list']
               }
               json resp
             end
 
             get '/list' do
-              json Major.all.map {|m| m.to_v1}.uniq
+              json Major.all.map { |m| m.to_v1 }.uniq
```

Comment:
why not `&:to_v1`?

Reply:
definitely should be, that rubocop rule must not be safe and so was not applied automatically. Feel free to make the changes, there were just a lot of them and I didn't want to do them all manually just yet (though we will need to do so eventually)